docs/references/backend/organization/get-organization-invitation.mdx

```diff
@@ -3,9 +3,11 @@ title: getOrganizationInvitation()
 description: Use Clerk's Backend SDK to retrieve an organization invitation.
```

Comment:
This page isn't in the manifest.

Reply:
handled [here](https://github.com/clerk/clerk-docs/pull/1253) - please review